libsel4camkes/docs/tls.md

```diff
@@ -0,0 +1,26 @@
+<!--
+     Copyright 2021, Data61, CSIRO (ABN 41 687 119 230)
+
+     SPDX-License-Identifier: BSD-2-Clause
+-->
+
+# Thread-Local Storage (TLS)
+
+This part of the library contains definitions and functions related to
+CAmkES-specific thread-local storage. These functions are intended for
+CAmkES-internal use to manage the state of each thread on top of the CAmkES
+runtime.
+
+## Usage
+
+When constructing templates for RPC-related CAmkES-connections, there may be
+situations where the seL4 endpoint reply cap may need to be managed. There are
```

Comment:
reply endpoint cap?

Reply:
in the seL4 API the cap is called `ReplyCap`, so "endpoint reply cap" is closer to that.